core/sql/executor/ExHdfsScan.cpp

```diff
@@ -1948,6 +1948,54 @@ short ExHdfsScanTcb::handleDone(ExWorkProcRetcode &rc)
   return 0;
 }
 
+void ExHdfsScanTcb::handleException(NAHeap *heap,
```

Comment:
Makes me wish we had a common base class for HBaseAccess, HdfsClient and maybe SequenceFileRead/Write. Any think that requiresJava/JNI to go to another file format. Could some of this logic be shared in that common base class? This is not a request for change, simply a question that will help me understand future refactoring choices.

Reply:
JavaObjectInterface is the base class for all these classes.